docs/development/event_policy_for_the_operator_de.md

```diff
@@ -34,6 +34,7 @@ erfolgreiche Installation, um einen guten Überblick über die Granularität des
 ![Bild mit Ereignissen für die erfolgreiche Installation des `postgresql` Dogu.](figures/events_without_errors.png)
 
 ## Verwendung von Ereignissen im `k8s-dogu-operator`
-
+<!-- TODO Please check this link because actual kubebuilder has ssl issues -->
+<!-- markdown-link-check-disable -->
 Die [kubebuilder Dokumentation](https://book-v1.book.kubebuilder.io/beyond_basics/creating_events.html) erklärt, wie man
```

Comment:
This part of the kubebuilder book was for unclear reasons deprecated or at least not continued in its current version. How about this link to the original book source? https://github.com/kubernetes-sigs/kubebuilder/commit/6d2f5e5978056f86b6d6ee18d6d319ae98f18ee9

I could not find a current version. I suggest to either use the link if the wording is so very precious. Otherwise I'd say let's link the event recorder in the golang packages involved:

- https://pkg.go.dev/k8s.io/client-go/tools/record#EventRecorder and 
- https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/cluster#Cluster